vendor/ember-cli-qunit/test-loader.js

```diff
@@ -1,18 +1,25 @@
 /* globals jQuery,QUnit */
 
 jQuery(document).ready(function() {
-  var TestLoaderModule = require('ember-cli/test-loader');
-  var TestLoader = TestLoaderModule['default'];
-  var addModuleIncludeMatcher = TestLoaderModule['addModuleIncludeMatcher'];
+  const TestLoaderModule = require('ember-cli/test-loader');
```

Comment:
I thought this was in the spirit of this article since we're importing something that is "class-like". Checking with the author :)

In any event, looks a case for good ol' var!